test/func/fixtures/jquery.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Fixture</title>
+  </head>
+  <body>
+    <div class="e2e-content"></div>
+    <div class="e2e-error"></div>
+    <script src="error-handler.js"></script>
+    <script src="../../../lib/little-loader.js"></script>
+    <script src="jquery/jquery.js"></script>
```

Comment:
If you don't mind, I'm going to keep it as is, because the rest of the func tests are organized like:

```
test/func/
  fixtures/
    SCENARIO.html
    SCENARIO/SCENARIO.js
  spec/
    SCENRIO.spec.js
```